tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/Core/InstrumentedTool.cs

```diff
@@ -46,6 +46,24 @@ public override async ValueTask<CallToolResult> InvokeAsync(RequestContext<CallT
             activity?.SetTag(TagName.ToolResponse, content);
             activity?.SetStatus(ActivityStatusCode.Ok);
 
+            if (result.Content?.FirstOrDefault() is TextContentBlock contentBlock)
```

Comment:
We should also have a corresponding update to the `MCPToolBase` for CLI invocation (though since CLI telemetry isn't currently working, it's a bit moot right now)